### DISCUSSION TITLE
challenge lesson 8 quite confusing for me

### DISCUSSION THREAD

**Discussion by xueyuanhuang** [Cyfrin/foundry-full-course-cu#2396] [General]
For challenge lesson 8-Foundry Fundamentals Section 3 NFTs
Only has this one single sentence
```
IPFS hash for website - ipfs://QmdCBj5B188qzXgG1LUCD6aKe6RJHqbSgu7YfPyncDirzh
```
I checked on the discussion, I guessing it should convert into something below, then it should automatically jump to some website?
```
https://ipfs.io/ipfs/QmdCBj5B188qzXgG1LUCD6aKe6RJHqbSgu7YfPyncDirzh
```
But it didnt work out, I am not sure what's the problem here.
Of course I try out different gateway, they all turn out as timeout.

if someone knows, pls help

**Comment by EngrPips** [comment]
Hello @xueyuanhuang, I believe when you load this `https://ipfs.io/ipfs/QmdCBj5B188qzXgG1LUCD6aKe6RJHqbSgu7YfPyncDirzh` in your browser, you are sending a request to the ipfs network asking for the resource that is represented by this URI `QmdCBj5B188qzXgG1LUCD6aKe6RJHqbSgu7YfPyncDirzh` if the network could not find the resource due to the fact that the node that has it isn't active then you won't get anything back.

**Reply by xueyuanhuang** [reply]
hi @EngrPips , so in this case I won't be able to mint the nft?🥲

**Reply by EngrPips** [reply]
Are you minting the `NFT` from the ipfs link?

**Reply by xueyuanhuang** [reply]
> Are you minting the `NFT` from the ipfs link?

yes, that's the hint ig

**Reply by EngrPips** [reply]
Is this `0xf215a0b6DD88d6029b5385D6fab51968337E963D` the contract address for the challenge?

**Reply by xueyuanhuang** [reply]
Not sure how you find this contract, but according to the repo, ipfs hash is the only hint
<img width="1440" alt="Snipaste_2024-09-05_18-58-02" src="https://github.com/user-attachments/assets/2b1dc544-7d17-437e-b151-3a9d5517551e">

**Reply by EngrPips** [reply]
Read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018

**Reply by xueyuanhuang** [reply]
> this

Oh that's the one I always ignore, thanks!

**Reply by EngrPips** [reply]
You are welcome, Friend.

**Reply by xueyuanhuang** [reply]
after I checked for others answer seems like previous hint is 
```
QmUgc1xpNgTMwSF2GBS5FTFKmSdrmSnLJD5DoYZtcQ2n5X
```
however currently in github repo the hint becomes
```
IPFS hash for website - ipfs://QmdCBj5B188qzXgG1LUCD6aKe6RJHqbSgu7YfPyncDirzh
```
it's not the same IPFS hash, is that means current repo has issue?

finally I am able to solve this challenge, however I think the hint totally misled me, no one is gonna to solve it without checking discussion....

**Reply by EngrPips** [reply]
I think the course got updated, and if the whole repo of the course is deployed to ipfs, it makes sense that its URI changes since its content changes. Anyway, I am glad you resolved it, and you solved the challenge already.

**Reply by xueyuanhuang** [reply]
@EngrPips maybe you're right, thanks bro.
and
@PatrickAlphaC what do you say about this, are you gonna update the content? As from my perspective, it's no way for new comers to solve this without any help :)

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
